exchanges/request/request.go

```diff
@@ -385,14 +385,6 @@ func isVerbose(ctx context.Context, verbose bool) bool {
 		return true
 	}
 
-	val := ctx.Value(contextVerboseFlag)
-	if val == nil {
-		return false
-	}
-
-	isCtxVerbose, ok := val.(bool)
-	if !ok {
-		return false
-	}
+	isCtxVerbose, _ := ctx.Value(contextVerboseFlag).(bool)
```

Comment:
I don't know why you introduced changes here, but there is no guarantee that the value is a bool when casting, please re-introduce `ok`

```suggestion
isCtxVerbose, ok := ctx.Value(contextVerboseFlag).(bool)
	if !ok {
		return false
	}
```